### DISCUSSION TITLE
[Bug] 8.6.0  Service name duplicate data

### DISCUSSION THREAD

**Discussion by colincatsu** [apache/skywalking#8061] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-rocketbot-ui)

### What happened

![image](https://user-images.githubusercontent.com/1275161/140013965-0fb732d4-f65c-4221-b853-f773017ec3e9.png)

sevice name not unique value ，There will be duplicate data

服务名字重复，运行一段时间后，发现很多服务名字重复了，我们请求两不是很大

### What you expected to happen

![image](https://user-images.githubusercontent.com/1275161/140014207-4ba975bd-990b-4459-b18a-53a62097d815.png)

### How to reproduce

有一段时间我们elastic search 密码改了，改回来后就出现服务名重复的现象

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Please update all contents to English for all readers.

Duplicate name thing has been explained repeatedly. There is no point to submit duplicate question.
